Mathlib/Tactic/FindSyntax.lean

```diff
@@ -84,11 +84,15 @@ elab "#find_syntax " id:str d:(&" approx")? : command => do
       match_results := match_results.insert mod <| (match_results.findD mod #[]).push (nm, rem.trim)
   -- We sort the messages to produce a more stable output.
   -- let msgsToString ← msgs.mapM (·.toString)
-  let mods := (match_results.toList).map fun (mod, msgs) =>
+  let sorted_results := match_results.toArray.qsort fun (mod1, _) (mod2, _) =>
+    (mod1.toString < mod2.toString)
```

Comment:
I think you can sort names directly without converting to string?

Reply:
Yes, `mod1.lt mod2`

Reply:
I used the `Name` sorting.  It is slightly different than `String` sorting, but given that arguably the "correct" sorting for this would be an "import-based" sorting, I am happy with the new ordering.